cdc/processor/sinkmanager/manager.go

```diff
@@ -596,9 +641,18 @@ func (m *SinkManager) GetTableStats(tableID model.TableID) (pipeline.Stats, erro
 		StartTs:  resolvedMark - 1,
 		CommitTs: resolvedMark,
 	}
-	err := m.sortEngine.CleanByTable(tableID, cleanPos)
-	if err != nil {
-		return pipeline.Stats{}, errors.Trace(err)
+	gcEvent := &gcEvent{
+		tableID:  tableID,
+		cleanPos: cleanPos,
+	}
+	select {
+	case m.engineGCChan <- gcEvent:
+	default:
+		log.Debug("Failed to send GC event to engine GC channel, engine GC channel is full",
```

Comment:
How about change the log level to `Warn`?

Reply:
I guess it will be too annoying.

Reply:
If they are dropped, we must have some ways to know it and then we can fix it.